core/adapters/eth_tx_test.go

```diff
@@ -176,9 +168,9 @@ func TestEthTxAdapter_Perform_ConfirmedWithBytesAndNoDataPrefix(t *testing.T) {
 			return nil
 		})
 	ethMock.Register("eth_blockNumber", utils.Uint64ToHex(sentAt))
-	receipt := models.TxReceipt{Hash: hash, BlockNumber: cltest.Int(confirmed)}
+	safe := sentAt - store.Config.MinOutgoingConfirmations()
```

Comment:
**Optional**

This line originally threw me off, because conceptually it's impossible for something to be `safe` **before** something that's `sent`. I later discovered you're using the receipt's `BlockNumber` to gauge whether something is safe so this works out.

If possible, I would recommend renaming `sentAt` to something else, perhaps `currentHeight`?

I would also rename the test `TestEthTxAdapter_Perform_ConfirmedWithBytesAndNoDataPrefix` from using the language Confirmed to using the more appropriate language **Safe**: `TestEthTxAdapter_Perform_SafeWithBytesAndNoDataPrefix`